packages/issuer/contracts/Issuer.sol

```diff
@@ -7,22 +7,21 @@ import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol
 import "./Registry.sol";
 
 contract Issuer is Initializable, Ownable {
-    event NewCertificationRequest(address indexed _owner, uint256 indexed _id);
+    event NewCertificationRequest(address indexed _owner, uint256 indexed _id, string indexed _deviceId);
     event ApprovedCertificationRequest(address indexed _owner, uint256 indexed _id, uint256 indexed _certificateId);
+    event RevokedCertificationRequest(address indexed _owner, uint256 indexed _id);
 
 	event CommitmentUpdated(address indexed _owner, uint256 indexed _id, bytes32 _commitment);
 	event MigrateToPublicRequested(address indexed _owner, uint256 indexed _id);
 	event PrivateTransferRequested(address indexed _owner, uint256 indexed _certificateId);
 	event CertificateMigratedToPublic(uint256 indexed _certificateId, address indexed _owner, uint256 indexed _amount);
+    event RevokedCertificate(uint256 indexed _certificateId);
```

Comment:
Ideally we would keep the naming scheme such as: 

"All events should be represented as verbs in the past tense such as CustomerRelocated, CargoShipped, etc."